packages/docs/docs/lambda/lambdarolepermissions.md

```diff
@@ -0,0 +1,102 @@
+---
+image: /generated/articles-docs-lambda-lambda-role-permissions.png
+id: lambdarolepermissions
+title: Lambda permissions
```

Comment:
There is already a page called Lambda permissions. Also, I think this tutorial should only be followed if you cannot follow the normal tutorial. Maybe "Using Lambda without IAM" is a better title?

Reply:
changed to 'Using Lambda without IAM user' to be more specific.